### ISSUE TITLE
Create community guidelines and code of conduct

### ISSUE THREAD

**Issue by spestana** [Ribbit-Network/organization#12]
I think Ribbit Network could have some set of community guidelines or a code of conduct we ask contributors to follow. This could include both details on how we organize work (such as how we expect issues, PRs to be submitted and formatted), and a code of conduct on how we work with each other (where best to have certain discussions, such as on discord versus on github; and anti-harassment policies). 

I am thinking of similar guidelines and codes of conduct I've seen, such as examples from the [Ada Initiative](https://adainitiative.org/) (and [example here](https://geekfeminism.fandom.com/wiki/Conference_anti-harassment/Policy)), this [opensource.guide](https://opensource.guide/code-of-conduct/) from github, and from [university sponsored hackathon events](https://snowex-hackweek.github.io/website/norms/CoC.html). I'm not sure how this works with Ribbit Network being a 501c organization... I am not a lawyer ;) but did want to get this discussion started.

**Comment by keenanjohnson** [comment]
Yes I think this would be an excellent idea and improvement @spestana ! [We have contributing guidelines here](https://github.com/Ribbit-Network/ribbit-network-frog-sensor/blob/main/CONTRIBUTING.md), but we don't address a lot of things in it, so it could definitely be improved and unified (one doc vs separate ones per repo, etc)

For example, we don't have a code of conduct or anti-harassment, etc.

The non-profit status doesn't affect anything about code of conduct, so I think we should feel free to draft whatever and run with it based on best practices from some of those examples you linked!

Thanks for starting this thread.

**Comment by spestana** [comment]
I will draft up something for anyone interested to review together.

**Comment by spestana** [comment]
Draft is here: https://github.com/spestana/ribbit-network-faq/blob/main/CODE_OF_CONDUCT.md
